docs/reference/query-dsl/vector-queries.asciidoc

```diff
@@ -0,0 +1,36 @@
+[[vector-queries]]
+== Vector queries
+
+Vector queries are specialized queries that work on vector fields to efficiently perform <<semantic-search,semantic search>>.
+
+<<query-dsl-knn-query,`knn` query>>::
+A query that finds the _k_ nearest vectors to a query vector for <<dense-vector,`dense_vector`>> fields, as measured by a similarity metric.
+
+<<query-dsl-sparse-vector-query,`sparse_vector` query>>::
+A query used to search <<sparse-vector,`sparse_vector`>> field types.
+
+<<query-dsl-semantic-query,`semantic` query>>::
+A query that allows you to perform semantic search on <<semantic-text,`semantic_text`>> fields.
+
+=== Deprecated vector queries
```

Comment:
@leemthompo , WDYT? I think adding an additional WARN is overkill, but double checking.

Reply:
Note-perfect as always @carlosdelest 👍